src/storyblok/Project.astro

```diff
@@ -0,0 +1,139 @@
+---
+import { renderRichText } from '@storyblok/astro';
+import { Image } from 'astro:assets';
+import StoryblokComponent from '@storyblok/astro/StoryblokComponent.astro'
+
+const { blok, name, projects } = Astro.props;
+
+import experiments from '../../src/images/svgs/experiments.svg';
+import stunts from '../../src/images/svgs/stunts.svg';
+import tools from '../../src/images/svgs/tools.svg';
+
+const fallback = {
+  experiments,
+  stunts,
+  tools
+}[blok.type.toLowerCase()] || experiments;
+
+function renderContent(content) {
+  return content ? renderRichText(content) : '';
+}
+---
+
+<article class="project">
+
+  <!-- Project Title -->
+  <h1>{name}</h1>
+
+  <!-- Project Logo or Fallback Image -->
+  {blok.logo ? (
+    <Image src={blok.icon.filename} inferSize={true} alt="" />
```

Comment:
Should this be `blok.logo` instead of `blok.icon`?

Reply:
Yes, thanks for catching that!